content/purgatorio/collect-and-prepare-data/data-collection.md

```diff
@@ -0,0 +1,209 @@
+
+Details are omitted and focus is drawn to the current subject (Data Collection) we are talking about. But this is the "big picture" to also keep in mind when working through individual steps.
+
+[Back to the index](#index)
+
+### Research or business questions
+
+The first thing before or during the Data Collection process will be asking the right question(s). The [right business questions](https://www.datapine.com/blog/data-analysis-questions/) is hard, but it has the [biggest impact](https://towardsdatascience.com/start-your-data-exploration-with-questions-2f1d42cff29e) on your performance of solving a particular problem. Remember, you want to [solve a problem](http://www.informit.com/articles/article.aspx?p=2271188&seqNum=2), not to create new ones! 
+
+And same goes for asking the right research question. If you intend to establish something based on a hunch or curiosity or other research then there has to be some good base to start with.
+
+You data collection process or step is entirely dependent on the research or the business problem you are trying to solve.
+
+At this point it's also important to understand that the end-model you are on the quest to create, is a direct reflection of these two factors:
+
+- Data (or datasets used)
+- Method (algorith(s) used)
+
+Although the second part i.e. Method will come a bit later, the first part is the primary or core component of the end-model. Methods can be swapped and at times the end-results may not differ by much but data is the key to getting a good model, otherwise you immediately can see the effects of GIGO (Garbage In, Garbage Out).
+
+When coming up with a question or a problem statement, it's also necessary to know the definition of "done" and use examples if possible to establish what the end-results should/could be comparable to once we reach the end point of the process (each iteration or the last iteration).
+
+[Back to the index](#index)
+
+### Start Small
+
+It's not efficient to try to handle Gigabytes or Petabytes of data each time you want to create a dataset as part of the Data Collection process. Just use [small subsets](https://sdtimes.com/bi/data-gets-big-best-practices-data-preparation-scale/) of the data (but take care that the data is representative and you know the problems or nuances it contains). Once you are certain about your process and the end-to-end flow has matured and stabilised you can consider using bigger chunks of data for the new iterations.
+
+[Back to the index](#index)
+
+### Small steps
+
+Just like the previous section [Start Small](#Start-small), starting small and staying small or growing slowly (or incrementally) is the most optimum way to proceed with Machine Learning or Deep Learning problem solving - given there are so many variables and moving parts. It's best to minimise the moving targets and pin them or fix them to a minimum or a fixed constant when working on any part of the end-to-end flow.
```

Comment:
I'd exlude the "a fixed constant" part in the last row, too much "OR" without commas, and it's clear anyway (imo)

Reply:
Yea agreed! It's a bit overloaded with those words!